app/assets/javascripts/bulk_updates.coffee

```diff
@@ -25,35 +41,37 @@ $(document).ready ->
 
         # matchingKeywords = $(keywordList).find('li').filter ->
         #   this.childNodes[0].nodeValue.trim() == value
-        keywordLengthMinimum = 2
+        keywordLengthMinimum = if type == 'science' then 2 else 1
         if splitKeywords.length > keywordLengthMinimum
-          fieldsToPopulate = ['category', 'topic', 'term', 'variablelevel1', 'variablelevel2', 'variablelevel3', 'detailedvariable']
+          scienceKeywordFields = ['category', 'topic', 'term', 'variablelevel1', 'variablelevel2', 'variablelevel3', 'detailedvariable']
+          locationKeywordFields = ['location_category', 'type', 'subregion_1', 'subregion_2', 'subregion_3', 'detailed_location']
+
+          fieldsToPopulate = if type == 'science' then scienceKeywordFields else locationKeywordFields
 
           $.each fieldsToPopulate, (index, value) ->
-            $('#new_' + value).val('')
+            $("#new-#{type}-keyword-container #new_#{value}").val('')
 
           $.each splitKeywords, (index, value) ->
-            $('#new_' + fieldsToPopulate[index]).val(value.trim())
-            $('#new_' + fieldsToPopulate[index]).trigger 'change'
+            $("#new-#{type}-keyword-container #new_#{fieldsToPopulate[index]}").val(value.trim())
+            $("#new-#{type}-keyword-container #new_#{fieldsToPopulate[index]}").trigger 'change'
 
     resetPicker = ->
       # Reset picker to top level
-      $('.select-science-keyword').attr 'disabled', true
+      $('.select-science-keyword, .select-location-keyword').attr 'disabled', true
       picker.resetPicker()
 
-    $('.selected-science-keywords').on 'click', '.remove', ->
-
     # Functions to validate user's ability to add keywords
     # Validate when user clicks on on item selection
     checkSelectionLevel = ->
-      selectionLevel = $('.eui-item-path li').length
+      type = picker.options.data_type
+      selectionLevel = $(".#{type}-keywords-picker .eui-item-path li").length
 
       # science keywords must be at least 3 levels deep
-      selectionMinimum = 3
+      selectionMinimum = if type == 'science' then 3 else 2
```

Comment:
We had discussed setting this as an option on the picker, I think that would be a meaningful improvement.